bundles/framework/hierarchical-layerlist/service/LayerlistExtenderService.js

```diff
@@ -0,0 +1,424 @@
+/**
+ * @class Oskari.framework.bundle.hierarchical-layerlist.OskariEventNotifierServic
+ */
+(function(Oskari) {
+    var _log = Oskari.log('HierarchicalLayerlist.LayerlistExtenderService');
+
+    Oskari.clazz.define('Oskari.framework.bundle.hierarchical-layerlist.LayerlistExtenderService',
+
+        /**
+         * @method create called automatically on construction
+         * @static
+         */
+        function() {
+            var me = this;
+            this.sb = Oskari.getSandbox();
+            this._options = {
+                /* Default options */
```

Comment:
Are these options for jstree?

Reply:
Yes, these are jstree options.

Reply:
Maybe add a comment or rename variable to clarify? Many files have similar structure where these kind of options (and state) are those of the Oskari class.